locale/en_US/manager.po

```diff
@@ -2514,8 +2514,11 @@ msgstr "Description for {$stat}"
 msgid "stats.editorialActivity"
 msgstr "Editorial Activity"
 
-msgid "stats.suppFileViews"
-msgstr "Supplementary File Views"
+msgid "stats.file.type.fullText"
+msgstr "Full Text"
```

Comment:
Instead of "full text" let's say "Primary File" and "Supplementary File". Not all primary files are necessarily text.